app/_src/gateway/upgrade-v2.md

```diff
@@ -60,7 +60,7 @@ The following table outlines various upgrade path scenarios to {{page.kong_versi
 
 {% endif_version %}
 
-{% if_version gte: 3.2.x %}
+{% if_version eq: 3.2.x %}
```

Comment:
`eq` will hide the upgrade path table of `3.2.x` from this page, as current version is `3.3.x`.

Is this expected?

Reply:
@outsinre Yes, since all the upgrade paths are copy/pasted in the 3.3 table, I just added a row for upgrades from 3.2.x and added an upgrade step to each row for 3.3